### DISCUSSION TITLE
For 900 message it is consuming 900 MB

### DISCUSSION THREAD

**Discussion by bugsfind900** [php-amqplib/php-amqplib#1079] [Q&A] [answered]
Hi ,
For each and every message php memory is increasing.
For example:- 
php memory limit is 900 Mb. 
once i started to consume 900 message . 
PHP memory reach at 900 MB and consume went shutdown.
Is there any way where can free memory after each message consume?

**Comment by lukebakken** [comment, marked as answer]
More than likely you have a bug in your program. Without a way to reproduce this issue, we can't assist you, so provide some code we can run to see the memory use increase. Thanks!
